hack/tools/tools.go

```diff
@@ -23,10 +23,11 @@ package tools
 import (
 	_ "github.com/drone/envsubst/v2/cmd/envsubst"
 	_ "github.com/joelanford/go-apidiff"
-	_ "github.com/mikefarah/yq/v4"
 	_ "github.com/onsi/ginkgo/ginkgo"
 	_ "gotest.tools/gotestsum"
 	_ "k8s.io/code-generator/cmd/conversion-gen"
+	_ "sigs.k8s.io/cluster-api"
+	_ "sigs.k8s.io/cluster-api/test/infrastructure/container"
```

Comment:
```suggestion
```
I don't think we have to add them here. We only have to add binaries, because if we wouldn't they would be dropped from the go.mod file. But those 2 dependencies will be part of go.mod because they are used in tilt-prepare.